lms/templates/user_dropdown.html

```diff
@@ -13,8 +13,29 @@
 
 from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
 %>
-
-% if uses_pattern_library:
+% if uses_bootstrap:
+    <li class="nav-item nav-item-hidden-collapsed">
+        <a href="${reverse('dashboard')}" class="menu-title">
+            <span class="sr-only">${_("Dashboard for:")}</span>
+            <% profile_image_url = get_profile_image_urls_for_user(self.real_user)['medium'] %>
+            <img class="user-image-frame" src="${profile_image_url}" alt="">
+        </a>
+    </li>
+    <li class="nav-item dropdown mt-2 nav-item-hidden-collapsed">
+        <% username = self.real_user.username %>
+        <a class="nav-link dropdown-toggle" id="navbarDropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">${username}</a>
+        <ul class="dropdown-menu dropdown-menu-right" id="${_("Usermenu")}" aria-labelledby="dropdownMenuLink" tabindex="-1">
+            <a class="dropdown-item" href="${reverse('dashboard')}">${_("Dashboard")}</a>
+            <a class="dropdown-item" href="${reverse('learner_profile', kwargs={'username': self.real_user.username})}">${_("Profile")}</a>
+            <a class="dropdown-item" href="${reverse('account_settings')}">${_("Account")}</a>
+            <a class="dropdown-item" href="${reverse('logout')}">${_("Sign Out")}</a>
+        </ul>
+    </li>
+    <li class="nav-item nav-item-open-collapsed-only"><a href="${reverse('dashboard')}">${_("Dashboard")}</a></li>
```

Comment:
It's a pity to duplicate these links three times. Maybe there should be an array with the user actions, and then all three places could iterate them,